src/TaylorSeries.jl

```diff
@@ -169,7 +171,8 @@ function numbr2str{T<:Real}(zz::Complex{T}, ifirst::Bool=false)
     return cadena
 end
 
-name_taylorNvar(n::Int) = string("⋅x", subscriptify(n))
+#name_taylorNvar(n::Int) = string("⋅x", subscriptify(n))
+name_taylorNvar(i::Int) = string(" ", _params_taylorN.variable_names[i])
```

Comment:
Since "⋅" (\cdot) is removed from the display of `TaylorN` variables, shouldn't we be consistent and do the same for `Taylor1` variables? (cf line 62)